src/Physics.hs

```diff
@@ -64,5 +45,24 @@ module Physics where
         Nothing -> game
     where
       (vx, vy) = ballVel game
-      c = wallsCollision (ballLoc game) ballRadius
-            (fromIntegral winWidth - wallWidth / 2) (fromIntegral winHeight - wallWidth / 2)
+      c = wallsCollision (ballLoc game) ballRadius gameWidth gameHeight
+
+  -- | Detect a collision with one a the bricks still present. Upon collisions,
+  --   update the bricks list
+  bricksBounce :: Game  -- ^ Initial game state
+               -> Game  -- ^ A new game state with an updated bricks
+  bricksBounce game =
+      case fst bc of
+        Nothing         -> game
+        Just TopSide    -> game {bricks = bricksUpdated, ballVel = (vx, -vy)}
+        Just BottomSide -> game {bricks = bricksUpdated, ballVel = (vx, -vy)}
+        Just LeftSide   -> game {bricks = bricksUpdated, ballVel = (-vx, vy)}
+        Just RightSide  -> game {bricks = bricksUpdated, ballVel = (-vx, vy)}
+      where
+      (vx, vy) = ballVel game
```

Comment:
This indentation of where clauses is at odds with the rest of the code. FWIW, I strongly suggest choosing and following a style guide.